src/main/java/org/opentripplanner/analyst/request/SampleGridRenderer.java

```diff
@@ -183,7 +188,7 @@ public WTWD closeSample(WTWD zUp, WTWD zDown, WTWD zRight, WTWD zLeft) {
                  * The computations below are approximation, but we are on the edge anyway and the
                  * current sample does not correspond to any computed value.
                  */
-                z.wTime = tMin + gridSizeMeters / v0;
+                z.wTime = tMin + gridSizeMeters / offRoadSpeed;
```

Comment:
I may come a bit late on this, but this change was rather important: there was a "bug" in the off-road speed vs speed along edge. The change was to distinguish between the two values (which are not the same in case of car routing for example) and use different names for the variables (v0 is not really meaningful -- this habit comes from my engineering background).